README.rst

```diff
@@ -1,6 +1,9 @@
 Flake8 Coding plugin
 =====================
 
+.. image:: https://img.shields.io/pypi/v/flake8-coding.svg
```

Comment:
Ok I'll remove it but I have it on all of my projects e.g. [flake8-comprehensions](https://github.com/adamchainz/flake8-comprehensions), [django-mysql](https://github.com/adamchainz/django-mysql) and it's used on many big projects such as [django-rest-framework](https://github.com/tomchristie/django-rest-framework).

People can easily end up at your repo page from google and then want to check the PyPI page / released version...